app/publishing/delinkedCalculation/get-unpublished.js

```diff
@@ -0,0 +1,28 @@
+const getUnpublishedDelinked = require('./get-unpublished-delinked')
+const updatePublished = require('./update-published')
+
+const getUnpublished = async (transaction) => {
+  const delinked = await getUnpublishedDelinked(transaction)
+
+  const unpublished = []
+
+  for (const item of delinked) {
+    if (item.calculationReference) {
+      unpublished.push({
+        ...item,
+        calculationId: item.calculationReference // Map calculationReference to calculationId
```

Comment:
Probably an obvious Q - what's the point of remapping the calculationId to be named calculationReference in getUnpublishedDelinked if we then just reset it back to calculationId here? couldn't we just leave it as called calculationId and then do `unpublished.push(item)`?

Reply:
(and obviously change the if to be looking for calculationId too)